xc/common/cmake/project_ray.cmake

```diff
@@ -201,20 +206,23 @@ function(PROJECT_RAY_PREPARE_DATABASE)
   foreach(PART ${PROJECT_RAY_PREPARE_DATABASE_PARTS})
     file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/channels/${PART})
     set(CHANNELS channels/${PART}/channels.db)
+    set(VPR_GRID_MAP channels/${PART}/vpr_grid_map.csv)
     add_custom_command(
-      OUTPUT ${CHANNELS}
+      OUTPUT ${CHANNELS} ${VPR_GRID_MAP}
       COMMAND ${CMAKE_COMMAND} -E env PYTHONPATH=${PRJRAY_DIR}:${symbiflow-arch-defs_SOURCE_DIR}/utils
       ${PYTHON3} ${FORM_CHANNELS}
       --db_root ${PRJRAY_DB_DIR}/${PRJRAY_ARCH}/
       --part ${PROTOTYPE_PART}
       --connection_database ${CMAKE_CURRENT_BINARY_DIR}/${CHANNELS}
+      --grid_map_output ${CMAKE_CURRENT_BINARY_DIR}/${VPR_GRID_MAP}
       DEPENDS
       ${FORM_CHANNELS}
       ${DEPS} ${DEPS2} ${DEPS3} simplejson progressbar2 intervaltree hilbertcurve
       ${PYTHON3} ${PYTHON3_TARGET}
       )
 
     add_file_target(FILE ${CHANNELS} GENERATED)
+    add_file_target(FILE ${VPR_GRID_MAP} GENERATED)
```

Comment:
Need to linearize these outputs. Make the target of vpr_grid_map.csv depend on the target of channels.db.

Reply:
Done